static/elements/chromedash-metadata.html

```diff
@@ -55,7 +55,7 @@
       if (detail) {
         // Came from an internal click.
         this.selected = Polymer.dom(e).localTarget.dataset.version;
-        this.fire('milestoneselect', {version: this.selected});
+        this.fire('queryChanged', {version: this.selected});
```

Comment:
prefer - case for events to be more consistent with native events.

`query-changed`